_content/excerpts/faith-in-awakening_geoff.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Faith In Awakening"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/books/PurityOfHeart/Section0005.html"
+drive_links:
+  - "https://drive.google.com/file/d/1BjYTgn4vfN20U5TZ49-Gf02-P-rnmJdq/view?usp=sharing"
+course: epistemology
+tags:
+  - faith
+  - psychology
+  - empiricism
+year: 2006
+booktitle: "Purity of Heart"
+pages: 10
+---
+
+> So there’s a tension in the Buddha’s recommendations about faith and empiricism. Few of Asian Buddhists I know find the tension uncomfortable, but Western Buddhists — raised in a culture where religion and faith have long been at war with science and empiricism — find it very disconcerting.
+
+This book chapter from [Purity of Heart](content/essays/purity-of-heart_geoff), discusses the delicate, but wonderful balance of faith and empiricism in Buddhism. Ṭhānissaro Bhikkhu highlights that it is direct seeing that liberates a practitioner, and faith operates as a working hypothesis. The chapter also focuses on the psychological importance of faith.
```

Comment:
That link is to chapter 1 of the book, not the book as a whole. The correct way to do this would be to add the book to `booklets` and link both these essays there via the `from_book` field. 😁

Reply:
Ah, sorry, I thought that was the whole book. I did the link correct, at least...right?
I can add the book to epistemology if you'd like...as a booklet since it's self-published, yes?

Reply:
Yeah! The link was done right 😁  No need to add the book I think. Let's just finish up the epistemology folder 😊